.github/workflows/release-deploy-all.yml

```diff
@@ -72,6 +72,7 @@ env:
   HOST_PREFIX: ${{ secrets.HOST_PREFIX }}
   BUILD_NAMESPACE: ${{ vars.BUILD_NAMESPACE }}
   KEYCLOAK_REALM: ${{ vars.KEYCLOAK_REALM }}
+  IS_FULLTIME_ALLOWED: ${{ vars.IS_FULLTIME_ALLOWED }}
```

Comment:
what about `repo-checks-tests.yml`? I see that E2Es are not failing but still we can add the default state of application from E2E perspective